### ISSUE TITLE
[springfox][swagger-ui] no model schema for POST objects

### ISSUE THREAD

**Issue by tckb** [springfox/springfox#978]
Hi, I used the swagger code-gen to generate server code for spring-mvc. The swagger -ui generated does not produce model schema.  

any workaround for it?
![screen shot 2015-09-20 at 8 45 36 pm]
(https://cloud.githubusercontent.com/assets/939542/9981365/b1657022-5fd8-11e5-87ab-56dc56a12119.png)

---
Update: The Issue was due to a bug in *JavaSpringMVC* template

Issue was resolved by adding

``` 
@RequestBody
```
to the body parameter.

**Comment by dilipkrish** [comment]
Could you share you're yaml/json that you pasted on to the editor. Keep in mind that the swagger-code gened libraries is a little bit behind on versions. You might want to up-rev the versions to atleast 2.2.2 and try again

**Comment by tckb** [comment]
> You might want to up-rev the versions to atleast 2.2.2 and try again

I am currently using the latest version. 

Hi dilipkrish, thanks for the reply. I fixed the issue. 
After careful investigations on <b>swagger-codegen</b>, I found the bug in the *JavaSpringMVC* template. The template was not appending 
``` 
@RequestBody
```
to the body parameter which resulted in the rendering issue. I fixed the template. Also, I found some more issues with the template. I'd have send it to the developers who maintain it.
